src/xdp/xsk.c

```diff
@@ -2815,12 +2950,12 @@ XskFillRingInfo(
     _Out_ XSK_RING_INFO *Info
     )
 {
-    ASSERT(Ring->Mdl != NULL);
+    // ASSERT(Ring->Mdl != NULL);
```

Comment:
More issues with MDL asserts (here and below)?

Reply:
Fixed